crates/ide/src/folding_ranges.rs

```diff
@@ -560,4 +563,16 @@ where
 "#,
         )
     }
+
+    #[test]
+    fn fold_return_type() {
+        check(
+            r#"
+fn foo()<fold returntype>-> (
+    bool,
+    bool,
+)</fold> { (true, true) }
+            "#,
```

Comment:
Can you also add a function to this fixture where the return type is on one line testing that it won't emit a folding range in that case? Basically like its done in the `fold_where_clause` test above

Reply:
Ah, I didn't think of that! Does what I added now check for the case you describe?

Reply:
yep, that seems good to me, thanks!
bors r+